engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/repositories.routes.js

```diff
@@ -56,6 +56,16 @@
                 parent: 'product.repositories'
             }
         })
+        .state('product.repository.advanced_sync', {
+            url: '/advanced_sync',
+            permission: 'view_products',
+            controller: 'RepositoryAdvancedSyncController',
+            templateUrl: 'products/details/repositories/details/views/repository-advanced-sync.html',
+            ncyBreadcrumb: {
+                label: "{{ repository.name }}",
```

Comment:
The label here should probably be `{{ 'Advanced Sync' | translate }}` because I am seeing Products > Animals > Repositories > Zoo > Zoo (i.e. the repo name repeated as the last breadcrumb).